### DISCUSSION TITLE
"Unplayable notes"

### DISCUSSION THREAD

**Discussion by ivan3002** [musescore/MuseScore#15350] [General]
Has Musescore 4 taken out the ability for notes highlighted in red to be played? I have written a concert Eb6 for Bb Trumpet but you don't hear it on the playback because it is deemed "unplayable".

**Comment by cbjeukendrup** [comment]
There is an issue about this: #13894 

This happens specifically with Muse Sounds; the available playback range is not very big yet (i.e. Muse Sounds instruments can't play very high or very low yet). If you switch to MS Basic, you should hear all notes. I expect the ranges for Muse Sounds will be extended in the future.

**Reply by ivan3002** [reply]
Awesome! Thanks for such a quick reply :)
